src/Services/ScrubberService.php

```diff
@@ -46,14 +46,23 @@ public static function autoSanitize(string &$jsonContent): void
                 ? Secret::decrypt($regexClass->getPattern())
                 : $regexClass->getPattern();
 
-            self::patternChecker($pattern, $jsonContent);
+            if (method_exists($regexClass, 'getReplacementValue')) {
```

Comment:
LGTM once https://github.com/AlexGodbehere/Laravel-Scrubber/pull/1 merges in and adds test coverage.

Reply:
Will do. Can you advise on the best approach to test this without stubbing a new class in the tests?

Reply:
Not really a good way, I created test coverage for you and made a PR into your branch. You will need to approve and merge it.

Reply:
Misread this. Nice work 🚀